libusb/os/linux_usbfs.c

```diff
@@ -2789,12 +2795,77 @@ static clockid_t op_get_timerfd_clockid(void)
 }
 #endif
 
+static int op_get_device_list(struct libusb_context * ctx,
```

Comment:
Since this is only used for Android, how about wrapping this function with #ifdef __ANDROID__/#endif? Otherwise, you will get the following warning on non-Android builds: warning: ‘op_get_device_list’ defined but not used